network/src/transport.rs

```diff
@@ -188,9 +188,12 @@ async fn upgrade_inbound<T: TSocket>(
     // try authenticating via noise handshake
     let (mut socket, remote_peer_id) = ctxt.noise.upgrade_inbound(socket).await.map_err(|err| {
         // security logging
-        warn!(security_log(security_events::INVALID_NETWORK_PEER)
-            .data_display("error", &err)
-            .field(network_events::NETWORK_ADDRESS, &addr));
+        warn!(
```

Comment:
same as other cases... I think when this is open to the public, we either need an aggressive sampler or counters.

Reply:
or we might as well disable it if this is the public network